commons/com.b2international.index/src/com/b2international/index/mapping/DocumentMapping.java

```diff
@@ -309,6 +309,11 @@ public boolean isSet(String field) {
 		final Class<?> fieldType = getFieldType(field);
 		return Set.class.isAssignableFrom(fieldType) || PrimitiveSet.class.isAssignableFrom(fieldType);
 	}
+	
+	public boolean isMap(String field) {
```

Comment:
Does `isObject` need to include maps? It is only used in `handleChangedInSourceAndTarget` which calls `handleObjectConflict`, and that sounds like it would/should work for this type as well: https://github.com/b2ihealthcare/snow-owl/blob/8.x/commons/com.b2international.index/src/com/b2international/index/revision/RevisionConflictProcessor.java#L129-L152

Reply:
Yes, good catch.

Reply:
This will be resolved in a separate PR.